ui/src/importer/importerTaskDetails/ImporterTaskDetailsComponents/ImporterReportHeader.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Grid, Icon } from 'semantic-ui-react';
+import { Link } from 'react-router-dom';
+import { CdsBackOfficeRoutes } from '../../../overridden/routes/BackofficeUrls';
+import { ImportedSearch } from './ImportedSearch';
+import { CancelImportTask } from './cancelImportTask';
+
+export class ImporterReportHeader extends React.Component {
+  render() {
+    const { isDisabled, isLoading, taskId, onSearch } = this.props;
+
+    return (
+      <Grid>
+        <Grid.Column floated="left" width={12}>
+          <Button
+            className="default-margin-top"
+            labelPosition="left"
+            icon="left arrow"
+            content="Import other files"
+            loading={isDisabled}
```

Comment:
Is this really necessary to be set? If so, `isLoading` shouldn't be the value to control it?

Reply:
isLoading is a bit different, i tried both and using isLoading is not the wanted behaviour

Reply:
the reason why i don't think i want to wrap my head around it too much